spec/models/course/level_spec.rb

```diff
@@ -6,7 +6,7 @@
 
   context 'before level_number is set' do
     it 'raises' do
-      expect { Course::Level.new.level_number }.to raise_error
+      expect { Course::Level.new.level_number }.to raise_error(RuntimeError)
```

Comment:
C# calls it InvalidOperationException, Java calls it IllegalStateException. I would call it something along those lines.